spec/features/admin/admin_articles_spec.rb

```diff
@@ -14,9 +9,10 @@
   end
 
   context "with session" do
-    let(:editor) { User.find_by(email: "pwhite@education.gov.uk") }
     scenario "visits article admin" do
-      login_as(editor)
+      create :article, title: "A typical support article", description: "And it's description", markdown: "For score and seven years ago..."
+      create :article, title: "Another support article", description: "And still another description", markdown: "How now brown cow?"
```

Comment:
Wish I could have persuaded you of the usefulness of FactoryGirl, it saves a lot of repetition, and any differences can be done as traits

Reply:
I am using FactoryGirl - the class name is not required.

Reply:
Better?